viz.yaml

```diff
@@ -137,6 +138,22 @@ process:
     scripts: [scripts/process/save_map.R]
     depends: 
       map_data: precip_cells
+  - 
+    id: precip_cell_data_json
+    location: cache/precip_cell_data.json
+    reader: json 
+    processor: precip_to_json
+    scripts: [scripts/process/precip_to_json.R]
+    depends: 
+      precip_data: precip_cell_data
+  -
+    id: precip_data_range
```

Comment:
i like this approach, and am also wondering if it's possible to go one step simpler with respect to the number of files we transfer. ideas:
1) could we tack on the precip_range list to precip_cell_data.json?
2) does map.js get templated (or could it be without too much effort) so that we could pop the numbers straight into that map.js file, so they're just part of the static site at publication time?

Reply:
OOOOO I don't know how #2 would work exactly, but that sounds exciting!